packages/lms/src/lib/components/navigation/Entity.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+    import { drawerStore } from '@skeletonlabs/skeleton';
+	import type { EntityMeta } from '@mollify/types';
+	export let entity: EntityMeta;
+	let open = false;
+    function drawerClose(): void {
+        drawerStore.close();
+    }
+    let typeOfEntity = entity.type;
+    
+    const entityIcons = {
+        'Assessment': 'Task',
+        'Lesson': 'Article',
+        'Module': 'Layers',
+        'Course': 'Book',
+        'Programme': 'Folder',
+        'Institution': 'School'
+    }
+    let icon = entityIcons[typeOfEntity] || '';
```

Comment:
Maybe have the default being a folder? as well as an alternative in the case where an entity has no further children to turn it into an article?